File: dev-docs/snippets/groups/remove-member.swift
import JsBaoClient

// Remove a member by userId. (To remove by email, Swift exposes a separate
// `removeMemberByEmail` method — see below.) Returns a typed `{ success: Bool }`.
func removeMember(client: JsBaoClient, groupType: String, groupId: String, userId: String) async throws {
  // #region example
  let result = try await client.groups.removeMember(
    groupType: groupType,
    groupId: groupId,
    userId: userId
  )
  let success = result.success
  // #endregion example
  _ = success
}

File: dev-docs/snippets/groups/remove-member.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Remove a member by userId. The same JS method is overloaded to also accept
// `{ email }` — e.g. `removeMember(groupType, groupId, { email })`.
export async function removeMember(
  client: JsBaoClient,
  groupType: string,
  groupId: string,
  userId: string,
) {
  // #region example
  const { success } = await client.groups.removeMember(groupType, groupId, userId);
  // #endregion example
  return success;
}
